eclair-core/src/main/resources/reference.conf

```diff
@@ -347,7 +348,7 @@ eclair {
       }
       safety-checks {
         // A set of basic checks on data to make sure we use the correct database
-        // Those checks are disabled by default because they wouldn't pass on a fresh new node with
+        // Those checks are disabled by default because they wouldnt pass on a fresh new node with
```

Comment:
The previous sentence was correct, this one isn't, revert?

Reply:
Sorry, I found out that, if copied entirely, your `reference.conf` leads to errors. And the error happens due to this symbol.
I will revert if you strongly disagree, of course.

Reply:
That's very surprising, I don't reproduce that...at least if you change it change it to proper english: `wouldn't` -> `would not`

Reply:
Fixed